pallets/msa/src/lib.rs

```diff
@@ -887,6 +888,18 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	/// Create Register Provider
+	pub fn create_registered_provider(
```

Comment:
Is this helper function is only for testing? Can we add a `#[cfg(test)] macro to this, or is there somewhere else it can be used?

Reply:
I am creating a PR for the main branch for this an additional things for testing.

Reply:
https://github.com/LibertyDSNP/frequency/issues/892